cmd/send-email/main.go

```diff
@@ -25,7 +28,18 @@ func main() {
         log.Fatal("Could not load config options.", err);
     }
 
-    err = email.Send(args.To, args.Subject, args.Body, false);
+    // If database cannot get course and using role based addressing
+    // Then, resolve users will return an error
+    // So, we do not error check on the get course method
+    course, err := db.GetCourse(args.Course);
+    emailTo, err := db.ResolveUsers(course, args.To);
+    if (err != nil) {
+        log.Fatal("Failed to resolve users:'%s', '%w'.", course.GetName(), err);
+    }
+    if (err != nil) {
+        log.Fatal("Could not resolve users.", err);
+    }
+    err = email.Send(emailTo, args.Subject, args.Body, false);
```

Comment:
Add a newline above this.
This is for a similar, but different reason as the whitespace between the if statements.
This is just to increase readability.
You want a reader to see visual blocks as doing one thing / part of the same unit.